LINUX/if_igb_netmap.h

```diff
@@ -159,6 +159,7 @@ igb_netmap_txsync(struct netmap_kring *kring, int flags)
 				// netmap_reload_map(pdev, DMA_TO_DEVICE, old_paddr, addr);
 			}
 			slot->flags &= ~(NS_REPORT | NS_BUF_CHANGED);
+			netmap_sync_map(na, (bus_dma_tag_t) na->pdev, &paddr, NETMAP_BUF_SIZE(na), NR_TX);
```

Comment:
It probably better to pass the actual size of of the packet, rather than ```NETMAP_BUF_SIZE(na)```. This was also the case for the rx path.